core/rs/core/src/commit.rs

```diff
@@ -37,5 +38,13 @@ pub unsafe fn commit_or_rollback_reset(ext_data: *mut crsql_ExtData) {
     let mut ordinals: mem::ManuallyDrop<Box<BTreeMap<Vec<u8>, i64>>> = mem::ManuallyDrop::new(
         Box::from_raw((*ext_data).ordinalMap as *mut BTreeMap<Vec<u8>, i64>),
     );
+
+    let mut table_infos = unsafe {
+        mem::ManuallyDrop::new(Box::from_raw((*ext_data).tableInfos as *mut Vec<TableInfo>))
+    };
     ordinals.clear();
+
+    for tbl_info in table_infos.iter_mut() {
```

Comment:
Should this also be cleared when `crsql_ensure_table_infos...` detects an schema change or is this cache still valid after the table schema changes?

Reply:
The cache is already cleared since `TableInfo` is re-generated